scripts/audit_settlement.py

```
@@ -0,0 +1,174 @@
+        client = ClobClient(
+            host="https://clob.polymarket.com",
+            key=key,
+            chain_id=137,
+            signature_type=sig_type,
+            funder=funder,
+        )
+        creds = client.create_or_derive_api_creds()
+        client.set_api_creds(creds)
+        res = client.get_balance_allowance(BalanceAllowanceParams(asset_type=AssetType.COLLATERAL, signature_type=sig_type))
+        return res
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)